sdk/typescript/docs/state-machine-engine.md

```
@@ -26,264 +26,194 @@ State changes trigger events that can be:
-**Purpose**: Standardizes and manages operation types:
-- **Standard Operations**: Pre-defined operation types (OWNERSHIP_TRANSFER, etc.)
-- **Custom Operations**: Support for custom operation types
-- **Workflow Generation**: Generate valid operation workflows
-- **Validation**: Ensure operations follow defined patterns
+### 1. Transaction Lifecycle (`TxStatus`)
 
-## State Transition Patterns
+The canonical status enum in `EngineBlox`:
 
-### 1. Transaction Lifecycle
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)